app/views/trips/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if @trip.errors.any? %>
+  <ul class="errors">
+    <% @trip.errors.each do |column, message| %>
+      <li>
+        <strong><%= column.capitalize %></strong> <%= message %>
+      </li>
+    <% end %>
+  </ul>
+<% end %>
+
+<%= form_with model: @trip, class: "edit_trip_form", url: passenger_trip_path do |f| %>
+
+  <p><%= "Form for editing a trip" %></p>
+
+  <%= f.label :date %>
+  <%= f.datetime_field :date %>
```

Comment:
Just a note, if I use the field to pick a date, I get a front-end validation error.